### PR TITLE
Fixes for highlights mobile

### PR DESCRIPTION
**Pull request by gabiborlea**
Use stacked buttons
Before
Screenshot 2022-03-24 at 14 30 22

After
Screenshot_1648720294

Fix get sessionId on mobile

### CODE REVIEW

**Review by saghul — APPROVED** [review]

**Comment by gabiborlea** [comment]
@saghul added some screenshots

**Comment by saghul** [comment]
Buttons should be stacked vertically, that's the common mobile UX pattern.

**Review by saghul — CHANGES_REQUESTED** [review]
(switching to request changes to we adapt the UI)

File: react/features/recording/components/Recording/styles.native.js [hunk 1]
```diff
@@ -42,7 +42,10 @@ const title = {
 const baseHighlightDialogButton = {
     borderRadius: BaseTheme.shape.borderRadius,
     height: BaseTheme.spacing[7],
-    flex: 1
+    flex: 1,
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center'
```
Comment (horymury) [inline react/features/recording/components/Recording/styles.native.js:48]: think you could replace:
`    flexDirection: 'row',
    justifyContent: 'center',
    alignItems: 'center'
`

with : justifyContent: 'space-around',

**Review by saghul — APPROVED** [review]

File: react/features/recording/functions.js [hunk 2]
```diff
@@ -215,7 +215,7 @@ export async function sendMeetingHighlight(state: Object) {
 
     const reqBody = {
         meetingFqn: extractFqnFromPath(state),
-        sessionId: conference.sessionId,
+        sessionId: conference.sessionId || conference.room.meetingId,
```
Comment (gabiborlea) [inline react/features/recording/functions.js:218]: @saghul is this a good way to get the sessionId on mobile?

Comment (saghul) [reply react/features/recording/functions.js:218]: Let me check.

Comment (saghul) [reply react/features/recording/functions.js:218]:
```suggestion
        sessionId: conference.getMeetingUniqueId(),
```

**Review by saghul — APPROVED** [review]

**Pull request merged**